start_db.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker compose up -d
```

Comment:
You could also add `docker-compose down -v` to remove previous containers and volumes before creating new container.

Reply:
It should be first command and then `docker-compose up -d`.

Reply:
I've added the command to stop the container before starting it up.